packages/modules/data-widgets/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "data-widgets",
+  "moduleName": "Data Widgets",
+  "version": "0.0.0",
```

Comment:
not 1.0.0? 😛

Reply:
It will auto bump (following Native approach)